main.py

```
@@ -0,0 +1,243 @@
+    try:
+        token_user = get_current_user(token)
+        logger.debug(f"Token user: {token_user}")
+        #await websocket.send_text(f"Welcome")
+        data = await websocket.receive_text()
+        logger.debug(f"Data received: {data}")
+        auth_data = json.loads(data)    
+        username = auth_data.get("username")
+        password = auth_data.get("password")
+        ws_user = authenticate_user(username, password)
+        if ws_user != token_user:
+            logger.error(f"Authentication error")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)